cpanfile

```diff
@@ -3,7 +3,7 @@ requires "DBD::SQLite";
 requires "DateTime::Format::ISO8601";
 requires "Digest::SHA";
 requires "FindBin";
-requires "Mojo::UserAgent";
```

Comment:
I think you should add Mojolicious as a prerequisite now that Mojo::UserAgent has been removed.